src/Controller/Api/PublicationController.php

```diff
@@ -138,7 +138,7 @@ public function postAction(Request $request, PubLinkUtil $pubLinkUtil)
 
         $doi = $request->request->get('doi');
 
-        if (false === preg_match('/^10\./', $doi)) {
+        if (false == preg_match('/^10\..*/', $doi)) {
```

Comment:
is there a reason for removing the type-hint?

Reply:
https://www.php.net/manual/en/function.preg-match.php -  PHPs nastiness - possible return of different types that both evaluate to false.

Reply:
It's only false if an error occurred. That seems unlikely to happen.
https://github.com/griidc/pelagos/commit/29af7783e0642b0b21e1e3f0edb4a0d9d0fc377b#diff-98d3078053a7ffd5bb3afaf91ae5dc5fL58-R58

Example: https://3v4l.org/UK5Mt